src-docs/src/views/facet/facet_example.js

```diff
@@ -69,6 +69,11 @@ export const FacetExample = {
       ),
       props: { EuiFacetGroup },
       demo: <FacetLayout />,
+      snippet: `// Restrict the width of default (vertical) if not restricted by parent
```

Comment:
```suggestion
      snippet: `{/* Restrict the width of default (vertical) if not restricted by parent */}
```

So it works in a copy paste in the middle of JSX

Reply:
Though again we probably want them to remove it. So maybe // is better.

Reply:
haha, Ok, I'll leave it